docs/source/logging.rst

```diff
@@ -85,7 +85,13 @@ The :func:`~~pytorch_lightning.core.lightning.LightningModule.log` method has a
 * `logger`: Logs to the logger like Tensorboard, or any other custom logger passed to the :class:`~pytorch_lightning.trainer.trainer.Trainer`.
 
 
-.. note:: Setting `on_epoch=True` will accumulate your logged values over the full training epoch.
+.. note::
+
+    -   Setting ``on_epoch=True`` will accumulate your logged values over the full training epoch.
```

Comment:
"accumulate" says `sum` to me, but I thought the default behavior is `mean`? Perhaps "reduce" is more appropriate?

Reply:
both are equally valid imo.

Reply:
the docs in this section do not mention the default reduction. 
We should say that the default is torch.mean